<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    background-color: rgb(0, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PTI picture SPCK/teacher.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>T_ _ _ her</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>eac</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>ace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>eca</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>750</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>ll</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>670</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>T_ b_e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>750</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>al</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PTI picture SPCK/table.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>750</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>la</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PTI picture SPCK/bag.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>ga</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>ag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_next3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightgreen;</string>
    </property>
    <property name="text">
     <string>Next lesson</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BacktoU1_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightgreen;</string>
    </property>
    <property name="text">
     <string>Back to unit1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>750</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>hoo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong8">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>750</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>oho</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PTI picture SPCK/school.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>670</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sc _ _ _ l</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrong7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>750</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;
    border: 1px solid black;
    padding: 5px;
    background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>hho</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
